rotkehlchen/tests/unit/decoders/test_shutter.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.chain.ethereum.modules.shutter.constants import (
+    CPT_SHUTTER,
+    SHUTTER_AIDROP_CONTRACT,
+)
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.constants.assets import A_ETH, A_SHU
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xCc60Eb2f64E7AD9b6924939B7985970D29A0108c']])
+def test_airdrop_claim(database, ethereum_inquirer, ethereum_accounts):
+    tx_hex = deserialize_evm_tx_hash('0x3f20929de51baefdc688997a05bde1e32120cb7d4e0fda5da3963b1a620d0a8b')  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash(tx_hex)
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    timestamp = TimestampMS(1706703155000)
+    gas_amount_str, claimed_amount = '0.006946508605618104', '1086.95652173913'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount_str)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burned {gas_amount_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=132,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_SHU,
+            balance=Balance(amount=FVal(claimed_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Claim {claimed_amount} SHU vested from shutter airdrop and deposit it to 0x22714964e6b9F17798A9e1AD3f2BAb87279876FC',  # noqa: E501
+            counterparty=CPT_SHUTTER,
+            address=SHUTTER_AIDROP_CONTRACT,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x36bE40f9bd613dFf47294E7e10D4cae072E06A2D']])
+def test_shu_delegation_from_self(database, ethereum_inquirer, ethereum_accounts):
```

Comment:
Remove one of these delegation tests. They are useless. You did not understand what delegation is.